nodes/constants.jl

```diff
@@ -0,0 +1 @@
+const LQR_gain_file = joinpath(@__DIR__, "..", "data", "lqr_gain.json")
```

Comment:
We should put any common constants we'll need across the various components into this file